poetry/core/masonry/builders/wheel.py

```diff
@@ -100,12 +107,15 @@ def build(self) -> None:
             with zipfile.ZipFile(
                 fd_file, mode="w", compression=zipfile.ZIP_DEFLATED
             ) as zip_file:
-                if not self._poetry.package.build_should_generate_setup():
-                    self._build(zip_file)
-                    self._copy_module(zip_file)
+                if not self._editable:
+                    if not self._poetry.package.build_should_generate_setup():
+                        self._build(zip_file)
+                        self._copy_module(zip_file)
+                    else:
+                        self._copy_module(zip_file)
+                        self._build(zip_file)
                 else:
-                    self._copy_module(zip_file)
-                    self._build(zip_file)
```

Comment:
I think we still need to build extensions (via the `build.py` script if it exists) so that they are available even in editable mode.

Reply:
Like this?

Reply:
It should be enough, yeah